project/Dependencies.scala

```diff
@@ -2,7 +2,7 @@ object Dependencies {
   // Should be sync with:
   //   - GA (.github/workflows/build-test.yml)
   //   - Mergify conditions (.mergify.yml)
-  val Scala212 = "2.12.15"
-  val Scala213 = "2.13.8"
-  val Scala3   = "3.1.2-RC3"
+  val Scala212 = "2.12.15" // sync! see comment above
+  val Scala213 = "2.13.8" // sync! see comment above
+  val Scala3   = "3.1.3-RC2" // sync! see comment above!
```

Comment:
I added a comment to each line because the comment above can not be seen in a GitHub diff.

Reply:
Thank you! 👍